internal/controllers/plans.go

```diff
@@ -176,31 +176,37 @@ func (s Server) AddPlanQuotaDefault(ctx echo.Context) error {
 	})
 }
 
+type quotaReq struct {
+	Username     string  `json:"user_name"`
+	ResourceName string  `json:"resourcetype_name"`
```

Comment:
Can we make the JSON key `resource_type_name`? It's more consistent with the naming conventions in our other services.